src/mlpack/core/data/dataset_mapper_impl.hpp

```diff
@@ -21,17 +21,16 @@ namespace data {
 
 // Default constructor.
 template<typename PolicyType>
-inline DatasetMapper<PolicyType>::DatasetMapper(const size_t dimensionality) :
-    types(dimensionality, Datatype::numeric)
+inline DatasetMapper<PolicyType>::DatasetMapper(const size_t dimensionality)
+    : types(dimensionality, Datatype::numeric)
```

Comment:
Yes, this is an issue in `clang-format` configuration. We have an option for configuring initializer list in clang-format [style guide](https://clang.llvm.org/docs/ClangFormatStyleOptions.html) i.e. `BreakConstructorInitializersBeforeComma`. I have already set it to`false` which matches with mlpack style guide but here I guess length of constructor is exceeding more than 80 chars, so it intends `:` to next line, not so sure why.